src/classes/project_data.py

```diff
@@ -49,10 +49,6 @@ def __init__(self):
         self.data_type = "project data"  # Used in error messages
         self.default_project_filepath = os.path.join(info.PATH, 'settings', '_default.project')
 
-        # Always set, in case the user creates the file mid-session
-        if info.USER_DEFAULT_PROJECT:
```

Comment:
This would always be true

Reply:
Yeah, originally I had the `if os.path.exists()` up here, but then I realized that was a bad idea for the reason given in that comment. But without the test, the rest of what was here became pointless, so I should've removed it completely.